samples/Localization.StarterWeb/Views/Shared/_Layout.cshtml

```diff
@@ -54,7 +54,7 @@
 
         <script src="//ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.min.js"></script>
         <script src="//ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/bootstrap.min.js"></script>
-        <script src="~/js/site.min.js" asp-append-version="true"></script>
+        <script src="~/js/site.js" asp-append-version="true"></script>
```

Comment:
The minimized version is empty and the unminified version is only like 3 lines long in the first place. @shanselman could you remove site.min.js while you're at it?